web/yaamp/modules/site/common_results.php

```diff
@@ -204,14 +204,14 @@ function cmp($a, $b)
 		<table>
 		<thead>
 		<tr>
-		<th data-sorter='numeric' align='left">PID</th>
-		<th data-sorter='numeric' align='left">Time</th>
-		<th data-sorter='numeric' align='left">Started</th>
-		<th data-sorter='numeric' align='left">Workers</th>
-		<th data-sorter='numeric' align='left">Port</th>
-		<th data-sorter='numeric' align='left">Symbol</th>
-		<th data-sorter='numeric' align='left">url</th>
-		<th data-sorter='numeric' align='left">fds</th>
+		<th data-sorter='numeric' align='left'>PID</th>
+		<th data-sorter='numeric' align='left'>Time</th>
+		<th data-sorter='numeric' align='left'>Started</th>
+		<th data-sorter='numeric' align='left'>Workers</th>
+		<th data-sorter='numeric' align='left'>Port</th>
+		<th data-sorter='numeric' align='left'>Symbol</th>
+		<th data-sorter='numeric' align='left'>url</th>
+		<th data-sorter='numeric' align='left'>fds</th>
```

Comment:
html single quotes are not standard use... and cant work echo '<td colspan='13'>';